meta-balena-common/recipes-core/initrdscripts/initramfs-framework_%.bbappend

```diff
@@ -70,7 +70,7 @@ RDEPENDS:initramfs-module-rootfs = "${PN}-base"
 FILES:initramfs-module-rootfs = "/init.d/90-rootfs"
 
 SUMMARY:initramfs-module-prepare = "Prepare initramfs console"
-RDEPENDS:initramfs-module-prepare = "${PN}-base os-helpers-logging os-helpers-fs"
+RDEPENDS:initramfs-module-prepare = "${PN}-base bash os-helpers-logging os-helpers-fs"
```

Comment:
@jakogut Is bash already a dependency for the initramfs because of resin-init-flasher?

Reply:
@alexgg Yes, and without this, we get a QA build failure 
```
QA Issue: /init.d/70-prepare contained in package initramfs-module-prepare requires /bin/bash, but no providers found in RDEPENDS:initramfs-module-prepare? [file-rdeps]
```